src/test/java/ch/astorm/jchess/util/ASCIIPositionRendererTest.java

```diff
@@ -9,17 +9,66 @@
 import org.junit.jupiter.api.Test;
 
 public class ASCIIPositionRendererTest {
+    private final String separator = System.lineSeparator();
+    private final String expectedOutput = "" +
```

Comment:
Can you please move the content in a file and load it with `ASCIIPositionRendererTest.class.getResourceAsStream(...)`.

Reply:
Moved